pkgs/nix-tools/darwin-rebuild.sh

```diff
@@ -116,6 +117,19 @@ while [ $# -gt 0 ]; do
       fi
       shift 1
       ;;
+    --offline)
+      extraMetadataFlags+=("$i")
+      extraBuildFlags+=("$i")
+      ;;
```

Comment:
```suggestion
```

Can you merge this with the option on line 47?

Reply:
Done!